docs/getting_started.rst

```diff
@@ -25,14 +25,7 @@ file, place it in the ``cfme_tests`` repository (along ``conftest.py``):
 
   #!/usr/bin/env bash
 
-  if hash dnf;
-  then
-    YUM=dnf
-  else
-    YUM=yum
-  fi
-
-  sudo $YUM install -y python-virtualenv gcc postgresql-devel libxml2-devel libxslt-devel zeromq3-devel libcurl-devel redhat-rpm-config gcc-c++ openssl-devel libffi-devel
+  pkcon install -y python-virtualenv gcc postgresql-devel libxml2-devel libxslt-devel zeromq3-devel libcurl-devel redhat-rpm-config gcc-c++ openssl-devel libffi-devel
```

Comment:
well, we don't necessarily care about debian,
but this removes the need for sudo, for example this works perfectly fine on a CSB without sudo enabled

and this can be put in a script that completely quickstarts a developer on red hat based distros